name 'MeTTaLog' is not defined
[()]
[()]
[(Error (llm (Agent <motto.agents.metta_agent.DialogAgent object at 0x7f5becd4e450>) (user "Hi")) Exception caught:
NameError: name 'MeTTaLog' is not defined

At:
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/examples/motto/agents/metta_agent.py(53): __call__
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/examples/motto/llm_gate.py(214): llm
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/examples/motto/llm_gate.py(240): <lambda>
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/atoms.py(346): execute
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/atoms.py(176): _priv_call_execute_on_grounded_atom
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/runner.py(31): run_step
  (17): run_step
)]
0.46user 0.07system 0:00.54elapsed 100%CPU (0avgtext+0avgdata 128988maxresident)k
0inputs+8outputs (0major+27863minor)pagefaults 0swaps
